pennylane/fermi/fermionic.py

```diff
@@ -108,8 +108,67 @@ def __repr__(self):
         r"""Terminal representation of a FermiWord"""
         return str(self)
 
+    def __add__(self, other):
+        """Add a FermiSentence, FermiWord or constant to a FermiWord. Converts both
+        elements to be added into FermiSentences, and uses the FermiSentence __add__
+        method"""
+
+        self_fs = FermiSentence({self: 1.0})
+
+        if isinstance(other, FermiSentence):
+            return self_fs + other
+
+        if isinstance(other, FermiWord):
+            other_fs = FermiSentence({other: 1.0})
+            return self_fs + other_fs
+
+        if isinstance(other, (float, int, complex)):
```

Comment:
What if the constant is a PL numpy array? e.g.,

`fermiword + np.array([1.0])`

Might consider using:

```
from numbers import Number
from pennylane.numpy.tensor import tensor

if isinstance(other, Number) or isinstance(other, tensor):
```